### DISCUSSION TITLE
How to get the camera pose file in json format

### DISCUSSION THREAD

**Discussion by yangrui12321** [NVlabs/instant-ngp#159] [Q&A] [answered]
In this project, the camera pose file in json format is required to be input, but the colmap output is in txt format such as images, cameras, etc. At the same time, the parameters are quaternions of QW, QX, QY, QZ, and their calculation methods are different. . How can I get files in json format? This question confuses me so much, thank you for your answers

**Comment by Tom94** [comment, marked as answer]
Hi, the python script `scripts/colmap2nerf.py` exists for this purpose.

Please read the [NeRF training & dataset tips](https://github.com/NVlabs/instant-ngp/blob/master/docs/nerf_dataset_tips.md) carefully.

**Comment by yangrui12321** [comment]
Thank you very much for your answer!
